.github/workflows/CI.yml

```diff
@@ -13,7 +13,9 @@ jobs:
 
       # Get local dependencies
       - run: npm install
-
+      # https://github.com/DefinitelyTyped/DefinitelyTyped/pull/49358/checks?check_run_id=1722333591#step:5:3341
+      - run: echo mongoose >> node_modules/@definitelytyped/dtslint-runner/expectedFailures.txt
```

Comment:
can you please revert the two CI changes in this PR too?

Reply:
@sandersn Thanks, done :heavy_check_mark: